### PR TITLE
Add Suppler/client button

### LINKED ISSUE #102: Client/Suppliers redirect button on navbar
need to add the button mentioned above.
#101

### PR DESCRIPTION
**Pull request by pazMenachem**
Added a button that will redirect the Supplier/Client to suppliers/client page.

Note - did modified the log in and sign up by creating a button for each.

Note - client redirection will be available and added once the clients page PR will be merged.
logged supplier
image

as guest
image
fix #102

### CODE REVIEW

File: static/css/nav_bar.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+.nav_sides{
+    display: flex !important;
+    align-items: center !important;
+    justify-content: flex-end !important;  
+    align-items: baseline !important;
+}
+
+.nav_side_element{
+    margin-right: 10px;
+}
```
Comment (thetaxas2468) [inline static/css/nav_bar.css:14]: new line

File: templates/buy_together_app/help_parts/nav.html [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
-            {% endif %}
+            <div class="container nav_sides">
+              {% if request.user.is_authenticated %}
+                <p class="nav_side_element">hello {{ request.user.username }}!</p>
+
+                {% if supplier %}
+                <form><a class="btn btn-primary nav_side_element" href="/suppliers" onclick="location.href='/my-page'">My Page</a></form>
+                {% endif %}
+
+                {% if client %}
```
Comment (thetaxas2468) [inline templates/buy_together_app/help_parts/nav.html:25]: In the context there is no client,if you log in as client i guess you will get only the request.user.name with out the tag <a>

File: templates/buy_together_app/help_parts/nav.html [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
-              <p style="color: beige;">hello {{ request.user.username }}!</p>
-            {% else %}
-              <a class="text-primary" href="/signup"><h5>Sign Up</h5></a>
-            {% endif %}
+            <div class="container nav_sides">
+              {% if request.user.is_authenticated %}
+                <p class="nav_side_element">hello {{ request.user.username }}!</p>
+
+                {% if supplier %}
+                <form><a class="btn btn-primary nav_side_element" href="/suppliers" onclick="location.href='/my-page'">My Page</a></form>
```
Comment (thetaxas2468) [inline templates/buy_together_app/help_parts/nav.html:22]: There is no need for a form in here,only <a> will do the work

File: templates/buy_together_app/help_parts/nav.html [hunk 4]
```diff
@@ -1,11 +1,36 @@
...
+                {% endif %}
+
+                {% if client %}
+                <form><a class="btn btn-primary nav_side_element" href="/clients" onclick="location.href='/my-page'">My Page</a></form>
```
Comment (thetaxas2468) [inline templates/buy_together_app/help_parts/nav.html:26]: There is no need for a form in here,only <a> will do the work

File: buy_together_app/views.py [hunk 5]
```diff
@@ -4,10 +4,14 @@
 from client.forms import ClientForm
 from supplier.forms import SupplierForm
 from django.db import IntegrityError
+from supplier.views import get_supplier
 
 
 def main_page(request):
```
Comment (yftacherzog) [inline buy_together_app/views.py:10]: Add tests.

File: templates/buy_together_app/help_parts/nav.html (hunk 2 above)
Comment (pazMenachem) [reply templates/buy_together_app/help_parts/nav.html:25]: ive just created a set up for the client, once the client page will be merged i will update it.

File: buy_together_app/views.py (hunk 5 above)
Comment (pazMenachem) [reply buy_together_app/views.py:10]: added

**Review by thetaxas2468 — APPROVED** [review]
Client should be deleted and changed when the client page is up,other than that

LGTM!

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**